tools/cli.py

```diff
@@ -411,6 +425,13 @@ def Main() -> None:
                 ('--leave_stopped', 'Leave the machine TERMINATED after '
                     'removing the service account (default: False)', False)
             ])
+  AddParser('gcp', gcp_subparsers, 's3togcs',
```

Comment:
Thoughts about renaming this to `s3toGSC`?

Reply:
Are you talking here about the capitalisation ?

Reply:
Yep